index.bs

```diff
@@ -347,7 +347,7 @@ try {
 }
 ```
 
-The {{CookieStore/set(name, value, options)|set()}} call above is a shorthand for the following:
+The {{CookieStore/set(options)|set()}} call above is a shorthand for the following:
```

Comment:
I think the link here should be to `{{CookieStore/set(name, value)|set()}}` ?

Reply:
And maybe it should say "shorthand for using an options dictionary, as follows:" ?

Reply:
SG, thanks for the catch.